### PR TITLE
DBZ-2219 Expanding support for MariaDB IF EXISTS/IF NOT EXISTS suppor…

### PR DESCRIPTION
**Pull request by jhiza**
…t on ALTER TABLE ddl

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jhiza, thanks! Can you take a look: there's a failure in one of the DDL parser tests.

File: debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4 [hunk 1]
```diff
@@ -591,12 +591,12 @@ alterView
 
 alterSpecification
     : tableOption (','? tableOption)*                               #alterByTableOption
-    | ADD COLUMN? uid columnDefinition (FIRST | AFTER uid)?         #alterByAddColumn
+    | ADD COLUMN? ifNotExists? uid columnDefinition (FIRST | AFTER uid)?         #alterByAddColumn
```
Comment (gunnarmorling) [inline debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4:594]: Could you add a comment that `ifNotExists` is specific to MariaDB? We haven't consistently marked those extensions so far, but I think we should begin doing so.

**Comment by jhiza** [comment]
> @jhiza, thanks! Can you take a look: there's a failure in one of the DDL parser tests.

Believe fixed

**Comment by gunnarmorling** [comment]
Still one related failure, something wrong with those comments, it seems. Can you please verify that this module builds locally before pushing? Thanks!

The other failure is unrelated and tracked via https://issues.redhat.com/browse/DBZ-2231.

**Comment by gunnarmorling** [comment]
Comments should be done using "// my comment...".

**Comment by jhiza** [comment]
> Comments should be done using "// my comment...".

That's not how ANTLR comments work, at least in my experimentation.  I tried exactly that and got test failures.

**Comment by gunnarmorling** [comment]
Closing in favour of #1628, which applies the correct commenting syntax and fixes the compilation failure. Thanks!

**Pull request closed without merging**